_includes/footer.html

```diff
@@ -1,39 +1,86 @@
 <footer class="site-footer h-card" role="footer">
-  <data class="u-url" href="{{ "/" | relative_url }}"></data>
-
-  <div class="wrapper-full">
-
-    <h2 class="footer-heading">{{ site.title | escape }}</h2>
-
-    <div class="footer-col-wrapper">
-      <div class="footer-col footer-col-1">
-        <ul class="contact-list">
-          <li class="p-name">
-            {%- if site.author -%}
-              {{ site.author | escape }}
-            {%- else -%}
-              {{ site.title | escape }}
-            {%- endif -%}
-            </li>
-            {%- if site.email -%}
-            <li><a class="u-email" href="mailto:{{ site.email }}">{{ site.email }}</a></li>
-            {%- endif -%}
-        </ul>
+  <data class="u-url" href="{{ " / " | relative_url }}"></data>
+
+  <div class="container">
+    <div class="site-footer__info">
+
+      <img class="site-footer__csis-logo" src="{{ 'assets/images/logos/csis_logo_white.svg' | relative_url }}" alt="{{ Center for Strategic and International Studies }}" title="{{ Center for Strategic and International Studies }}" />
+
+      <p class="site-footer__desc">
+        <span class="site-footer__section-title"> {{site.data.language.footer.about_the_commission_heading }}</span>
+
+        {{site.data.language.footer.about_the_commission_description }}
+      </p>
+
+
+      <div class="site-footer__follow">
+        <span class="site-footer__follow-csis">Follow CSIS’s</span>
+        <span class="site-footer__follow-program">Global Health Program</span>
+        <i class="site-footer__follow-icon icon-twitter"></i>
       </div>
 
-      <div class="footer-col footer-col-2">
-        {%- include social.html -%}
+      <ul class="site-footer__contact-list">
+        <li>Contact</li>
+        <li><i class="icon-phone"></i>{{site.data.language.footer.phone_number }}</li>
+        <li><i class="icon-email"></i>{{site.data.language.footer.email_address }}</li>
```

Comment:
The email address should be a link to a `mailto` reference with the subject being the site's name.